source/_components/insteon_local.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: "Insteon(local)"
+description: "Instructions how to setup the Insteon Hub locally within Home Assistant."
+date:  2016-12-18 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: insteon.png
+ha_category: Hub
+ha_iot_class: "Local Polling"
```

Comment:
`ha_version:` tag will need to be added to all new files to reflect the version the component was added in.